src/index.test.ts

```diff
@@ -150,23 +149,10 @@ describe('Finalize transaction', () => {
       xPriv: xPrivString,
     });
 
-    const draftTransaction: DraftTransaction = JSON.parse(draftTxJSON2);
-    const transaction = spvWalletClient.SignTransaction(draftTransaction);
+    const draftTransaction: DraftTx = JSON.parse(draftTxJSON2);
+    const transaction = await  spvWalletClient.SignTransaction(draftTransaction);
     expect(typeof transaction).toBe('string');
   });
-
-  test('draftTxJSON2 error', async () => {
```

Comment:
Removed this specific test, as we are no longer veryfing tx with .verify() method (🧵  below).